packages/combo-box/test/external-filtering.test.js

```diff
@@ -38,7 +38,7 @@ describe('external filtering', () => {
       expect(getFocusedItemIndex(comboBox)).to.equal(-1);
     });
 
-    it('should focus on filtered value', () => {
+    it.skip('should focus on filtered value', () => {
```

Comment:
Is skipping here intentional as a consequence of this change? The test passes locally for me.